docs/docsite/sphinx_conf/core_conf.py

```diff
@@ -32,7 +32,7 @@
 # the repository version needs to be the one that is loaded:
 sys.path.insert(0, os.path.abspath(os.path.join('..', '..', '..', 'lib')))
 
-VERSION = 'devel'
+VERSION = '2.14'
```

Comment:
Is this correct to change devel to 2.14?

Reply:
Good question. So in this instance, yes, because if you notice way above, this PR is going directly into `stable-2.14` branch.